packages/jmix-react-ui/src/crud/list/ui-callbacks/usePaginationChangeCallback.ts

```diff
@@ -17,17 +17,20 @@ export function usePaginationChangeCallback<TEntity>(
       };
       saveHistory(routingPath, entityListState.pagination);
 
-      if (currentScreen.params === undefined) {
-        currentScreen.params = {}
-      }
+      if(currentScreen) {
```

Comment:
According to our code style we should use `if (something != null)` instead of either `if (something)` or `if (something !== undefined)`.